modules/hosted-control-planes-version-support.adoc

```diff
@@ -5,21 +5,20 @@
 
 :_content-type: CONCEPT
 [id="hosted-control-planes-version-support_{context}"]
-= Versioning support for hosted control planes
+= Versioning for hosted control planes
 
-With each major, minor, or patch version release of {product-title}, the following components of hosted control planes are released:
+With each major, minor, or patch version release of {product-title}, two components of hosted control planes are released:
 
 * HyperShift Operator
-* `HostedCluster` and `NodePool` API resources
 * Command-line interface (CLI)
 
-The HyperShift Operator manages the lifecycle of hosted clusters that are represented by `HostedCluster` resources. The HyperShift Operator is released with each {product-title} release. Any HyperShift Operator that is released with a minor _n_ version of {product-title} must support _n_, _n-1_, and _n-2_ minor versions of {product-title}.
+The HyperShift Operator manages the lifecycle of hosted clusters that are represented by `HostedCluster` API resources. The HyperShift Operator is released with each {product-title} release. Any HyperShift Operator that is released with a minor version of {product-title} must work with all versions of {product-title} where hosted control planes are available. However, because the hosted control planes feature was introduced as a Technology Preview in {product-title} version 4.11, it is available with versions only as far back as 4.11.
```

Comment:
>Any HyperShift Operator that is released with a minor version of {product-title} must work with all versions of {product-title} where hosted control planes are available. However, because the hosted control planes feature was introduced as a Technology Preview in {product-title} version 4.11, it is available with versions only as far back as 4.11.

Each HO n release must work with n-1,n-2 ocp versions. With the GA n release i.e 4.12 being the minimum supported.
4.12 will support only 4.12
4.13 will support 4.13 and 4.12
4.14 will support 4.14, 4.13 and 4.12
4.15 will support 4.15, 4.14 and 4.13.

Reply:
@enxebre The phrase "With GA n release i.e 4.12 being the minimum supported" is a confusing to me. Hosted control planes are still a Technology Preview feature for 4.12. As I understand it, that means they're not really "GA" in 4.12.

Reply:
@enxebre I just had a chat with Adel about a few doc-related items, and we talked about this one. He clarified your comment for me, and we decided to remove the line, "However, because the hosted control planes feature was introduced as a Technology Preview in {product-title} version 4.11, it is available with versions only as far back as 4.11."

He mentioned we'll need to be purposely vague here until version 4.15, when we can say something like, "4.15 HO supports 4.15, 4.14, and 4.13," as you already mentioned. 

When you can, please let me know if the latest updates look OK, and I can move this along to be merged. Thanks!

FYI @jiezhao16

Reply:
>Any HyperShift Operator that is released with a minor version of {product-title} must work with all versions of {product-title} where hosted control planes are available.

When I read this I understand that any version N of the HO must work with all Version < N of openshift which is not true (at least for now). Am I reading it right?

Reply:
>must work with all versions of {product-title} where hosted control planes are available.

Is this referring to the management cluster of to the versions of the HC.release?

Either way I don't think we should make that support statement.

Reply:
@enxebre Thanks for your feedback. Do you think it would be best at this point to remove the following sentence?

`Any HyperShift Operator that is released with a minor version of {product-title} must work with all versions of {product-title} where hosted control planes are available.`

Then, I can plan to update the docs when hosted control planes are GA to be more specific. It's tricky to document this right now since we're still in Technology Preview.